test/typescript-validate.ts

```diff
@@ -31,3 +36,40 @@ export async function issue282() {
 
   isString(authentication.token);
 }
+
+// https://github.com/octokit/auth-app.js/issues/603
+export async function issue603() {
+  createAppAuth({
+    appId: "1",
+    privateKey: "",
+  });
+
+  const options: StrategyOptions = {
+    appId: "1",
```

Comment:
make sure to not only test with a stringified number, but with that actually looks like a ClientID

```suggestion
    appId: "Iv1.0123456789abcdef",
```